Eatery/Controllers/Eateries/Campus/TabbedPageViewController.swift

```diff
@@ -25,6 +25,7 @@ protocol TabbedPageViewControllerDelegate: AnyObject {
 class TabbedPageViewController: UIViewController {
 
     let viewControllers: [UIViewController]
+    var hasLoaded: Bool = false
```

Comment:
Delete type annotation. Also, can these be `private`? Also I prefer `didLoad` to `hasLoaded`.